src/core/guid.js

```diff
@@ -1,6 +1,11 @@
+// Look up table for GUIDs
+const LUT = [];
+for (let i = 0; i < 256; i++) {
+    LUT[i] = (i < 16 ? '0' : '') + i.toString(16);
+}
+
 /**
  * @name guid
- * @namespace
```

Comment:
I'm curious why this PR removes this JSDoc tag.

Reply:
Ah, so this is a namespace. Fixed.